apps/navstar_l4lb/test/navstar_l4lb_ipvs_SUITE.erl

```diff
@@ -33,21 +33,21 @@ all(_, _) -> [].
 
 init_per_testcase(_, Config) ->
     "" = os:cmd("ipvsadm -C"),
-    os:cmd("ip link del minuteman"),
-    os:cmd("ip link add minuteman type dummy"),
+    os:cmd("ip link del navstar_l4lb"),
```

Comment:
i wonder, should we clean up the old minuteman dummy interface? maybe not in the code here, just in a script or something?